auditorium/views/main.js

```diff
@@ -60,16 +67,61 @@ function view (state, emit) {
     if (range.query || Object.keys(foreign).length) {
       url += '?' + new window.URLSearchParams(Object.assign(foreign, range.query))
     }
-    var anchor = html`<a href="${url}">${range.display}</a>`
+    var anchor = html`<a href="${url}" class="f5 b link dim ph3 pv2 mr2 mb1 dib br1 dark-green bg-black-05">${range.display}</a>`
     return html`
-      <li>
-        ${active ? html`<strong>${anchor}</strong>` : anchor}
+      <li class="mb1">
+        ${active ? html`<a href="${url}" class="f5 b link bb bw2 ph3 pv2 mr2 mb1 dib br1  dark-green bg-black-05">${range.display}</a>` : anchor}
       </li>
     `
   })
   var rangeSelector = html`
-    <h4>${__('Show data from the:')}</h4>
-    <ul>${ranges}</ul>
+      <h4 class ="f5 normal mt0 mb3">${__('Show data from the last')}</h4>
+      <ul class="flex flex-wrap list pl0 mt0 mb3">${ranges}</ul>
+    `
+
+  var goAccounts = html`
+      <h4 class ="f5 normal mt0 mb3">Manage accounts</h4>
+      <a href="/auditorium/account/" class="w-100-ns f5 tc link dim bn ph3 pv2 mr1 mb2 dib br1 white bg-dark-green">
+        Overview
+      </a>
+    `
+
+  if (isOperator) {
+    var rowRangeManage = html`
```

Comment:
This could be a lot less repetitive:

```js
var width = isOperator ? 'w-30-ns' : 'w-20-ns'
var firstCardContent = isOperator ? goAccounts : manage
var rowRangeManage = html`
    <div class="flex flex-column flex-row-ns mt4">
      <div class="${width} w-100 pa3 mb2 mr2-ns ba b--black-10 br2 bg-white-40">
        ${firstCardContent}
      </div>
      <div class="w-100 w-80-ns pa3 mb2 ba b--black-10 br2 bg-white-40">
        ${rangeSelector}
      </div>
    </div>
  `
```

Reply:
:+1: